deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -100,7 +96,12 @@ def collect_screenshots_for_languages(path)
         UI.important("Framed screenshots are detected! 🖼 Non-framed screenshot files may be skipped. 🏃") if prefer_framed
 
         language_dir_name = File.basename(lng_folder)
-        language = available_languages[language_dir_name.downcase] || language_dir_name
+
+        if available_languages[language_dir_name.downcase].nil?
+          UI.user_error!("#{language_dir_name} is not an available language. Please verify that your language codes are available in iTunesConnect. See https://developer.apple.com/library/content/documentation/LanguagesUtilities/Conceptual/iTunesConnect_Guide/Chapters/AppStoreTerritories.html for more information.")
```

Comment:
not sure i understand what you mean? showing a warning would still result in a crash right?